src/components/Dashboard/TodoList.vue

```diff
@@ -29,7 +29,7 @@
 
     <v-expansion-panels v-if="todoItems && todoItems.length > 0" accordion  v-model="panel">
       <v-expansion-panel
-        class="align-items-top todo-item"
+        class="align-items-top todo-item px-6 py-5"
```

Comment:
This layout change (and on line 41) was needed to get rid of weird special case whitespaces in sub-components. This is way cleaner!